projectroles/app_settings.py

```diff
@@ -42,7 +44,7 @@
     #: Example ::
     #:
     #:     'example_setting': {
-    #:         'scope': 'PROJECT',  # PROJECT/USER
+    #:         'scope': 'PROJECT',  # PROJECT/USER/SITE
```

Comment:
Just noticed that `PROJECT-USER` is missing from this example. Not your fault as it should have been there before, but let's add it :)